float64 roll_or_x
float64 pitch_or_y
float64 yaw
float64 thr_z
